test/graph/resolvers/event_resolver_test.rb

```diff
@@ -72,4 +72,98 @@
       assert_nil Event.find_by(id: id)
     end
   end
+
+  describe '.all' do
+    let(:organization) { organizations(:minimum) }
+    let(:event_type) { event_types(:minimum) }
+    let(:office) { offices(:san_francisco) }
+    let(:other_office) { offices(:madison) }
+    let(:event1) do
+      Event.create!(
+        organization: organization,
+        title: 'test1',
+        type: event_type,
+        starts_at: event1_start,
+        ends_at: event1_start + 1.hour,
+        capacity: 10,
+        location: 'somewhere',
+        office: office
+      )
+    end
+    let(:event2) do
+      Event.create!(
+        organization: organization,
+        title: 'test2',
+        type: event_type,
+        starts_at: event2_start,
+        ends_at: event3_start + 1.hour,
+        capacity: 10,
+        location: 'somewhere',
+        office: office
+      )
+    end
+    let(:event3) do
+      Event.create!(
+        organization: organization,
+        title: 'test3',
+        type: event_type,
+        starts_at: event3_start,
+        ends_at: event3_start + 1.hour,
+        capacity: 10,
+        location: 'somewhere',
+        office: other_office
+      )
+    end
+    let(:event1_start) { Time.new(2018, 1, 1, 1, 0) }
+    let(:event2_start) { Time.new(2018, 1, 1, 3, 0) }
+    let(:event3_start) { Time.new(2018, 1, 1, 2, 0) }
```

Comment:
I'd find this test a little easier to read if 1, 2, and 3 was the actual start time order. Then unsorted could be `[event2, event1, event3]`, asc would be `[event1, event2, event3]`, desc would be `[event3, event2, event1]`.